File: IOT/gurpreetk12_gurpreetkaur_2125cs1094_2/FunctionandArrays/SpanOfArray.cpp
#include<iostream>
using namespace std;

int main(){
    int n;
    cin>>n;
    int arr[n],e;
    for(int i=0;i<n;i++)
    {
        cin>>e;
        arr[i]=e;
    }
    int max=arr[0],min=arr[0];
    for(int i=0;i<n;i++)
    {
        if(arr[i]>=max)
        {
            max=arr[i];
        }
        if(arr[i]<=min)
        {
            min=arr[i];
        }
    }
    int sum=max-min;
    cout<<sum;
    
    
}

File: AR-VR/kapilsingh2003_KapilKumarSingh_2125cs1169_2/FunctionAndArrays/SpanOfArray.java
import java.io.*;
import java.util.*;

public class Main{

public static void main(String[] args) throws Exception {
    Scanner scn = new Scanner(System.in);
    int n = scn.nextInt();
    int[] arr = new int[n];
    for(int i=0; i<n; i++){
        arr[i]=scn.nextInt();
    }
    int max = arr[0];
    int min=arr[0];
    for(int i=0; i<n; i++){
        if(arr[i]>max)
        max=arr[i];
        if(arr[i]<min)
        min=arr[i];
    }
    int res=max-min;
    System.out.print(res);
 }

}